logging/examples_test.go

```diff
@@ -86,6 +87,23 @@ func ExampleClient_Logger() {
 	_ = lg // TODO: use the Logger.
 }
 
+func ExampleHTTPRequest() {
+	ctx := context.Background()
+	client, err := logging.NewClient(ctx, "my-project")
+	if err != nil {
+		// TODO: Handle error.
```

Comment:
Are these todos meant to be merged in?

Reply:
They are intentional. It looks to be a godoc convention among all of the godoc examples